packages/website/pages/components/[slug].tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import ErrorPage from 'next/error';
+import { remark } from 'remark';
+import html from 'remark-html';
+import { css } from 'emotion';
+
+import * as f36Components from '@contentful/f36-components';
+import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
+
+import { geltAllMDX, getPageBySlug } from '../../utils/content';
+
+const styles = {
+  root: css({
+    minHeight: '100vh',
+    padding: '0 0.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  }),
+};
+
+export default function ComponentPage(props: {
+  data: any;
+  content: string;
+  previews: string[];
+  slug: string;
+}) {
+  const router = useRouter();
+  if (!router.isFallback && !props.slug) {
+    return <ErrorPage statusCode={404} />;
+  }
+
+  return (
+    <div className={styles.root}>
+      <h1>{props.data.title}</h1>
+
+      {props.previews &&
+        props.previews.map((preview, idx) => (
+          <LiveProvider
+            key={idx}
+            scope={{ ...f36Components }}
+            code={transformCode(preview)}
+            language="tsx"
+          >
+            <f36Components.Box padding="spacingM">
+              <LivePreview />
+            </f36Components.Box>
+            <LiveEditor />
+            <LiveError />
+          </LiveProvider>
+        ))}
```

Comment:
I think this could be removed, this was an experiment on separating examples from the mdx files but we won't use this approach anymore

Reply:
I'm going to remove this in the follow up PR